### ISSUE TITLE
How do I get rotated MTIImage ?

### ISSUE THREAD

**Issue by PatrickSCLin** [MetalPetal/MetalPetal#20]
```
let device = MTLCreateSystemDefaultDevice()!
let option = MTIContextOptions()
let context = try! MTIContext(device: device, options: option)
let transformFilter = MTITransformFilter()
let angle = CGFloat(90 / 180 * Double.pi)
let tranform = CATransform3DMakeRotation(angle, 0, 0, 1)
transformFilter.transform = tranform
transformFilter.inputImage = MTIImage(cgImage: image.cgImage!, options: [.SRGB: false], alphaType: .alphaIsOne)
let outputImage = transformFilter.outputImage
```

assumed size of inputImage is 720 * 1280
I was expected that size of outputImage became  1280 * 720, but it's still same size of inputImage

How do I get rotated outputImage ?

**Comment by PatrickSCLin** [comment]
Can you add one more method for MTIImage ?

```
/* Returns a new image representing the original image transformeded for the given CGImagePropertyOrientation */
    @available(iOS 11.0, *)
    open func oriented(_ orientation: CGImagePropertyOrientation) -> CIImage
```

**Comment by YuAo** [comment]
added a `func oriented(_ orientation: CGImagePropertyOrientation) -> MTIImage` method on `MTIImage`. https://github.com/MetalPetal/MetalPetal/blob/master/Frameworks/MetalPetal/Filters/MTIImage%2BFilters.m#L22
